ci-operator/config/opendatahub-io/opendatahub-operator/opendatahub-io-opendatahub-operator-main.yaml

```
@@ -18,14 +18,23 @@ images:
   dockerfile_path: Dockerfiles/Dockerfile
   from: ubi
   to: opendatahub-operator
+- context_dir: .
+  dockerfile_path: Dockerfiles/rhoai.Dockerfile
+  from: ubi
+  to: opendatahub-operator-rhoai
 operator:
   bundles:
   - as: opendatahub-operator-bundle
     dockerfile_path: Dockerfiles/bundle.Dockerfile
     skip_building_index: true
+  - as: opendatahub-operator-rhoai-bundle
+    dockerfile_path: Dockerfiles/rhoai-bundle.Dockerfile
+    skip_building_index: true
   substitutions:
   - pullspec: REPLACE_IMAGE:latest
     with: pipeline:opendatahub-operator
+  - pullspec: REPLACE_IMAGE:latest-rhoai
+    with: pipeline:opendatahub-operator-rhoai
```

With this order, it will replace `REPLACE_IMAGE:latest` also for rhoai, which will be `replaced_image-rhaoi`.
It should work correctly putting before the `REPLACE_IMAGE:latest-rhoai` subst.

```suggestion
  substitutions:
  - pullspec: REPLACE_IMAGE:latest-rhoai
    with: pipeline:opendatahub-operator-rhoai
  - pullspec: REPLACE_IMAGE:latest
    with: pipeline:opendatahub-operator
```